index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Your description goes here">
+    	<meta name="keywords" content="one, two, three">
+
+		<title>Relaxr</title>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+		<div class="container">
+			<header>
+					<h1>Relaxr</h1>
+				<center>
+					<h2>Get piece of mind with a single tap</h2>
+					<a class="button download">Get it Now</a>
+				</center>
+			</header>
+			<div class="benefits">
+				<h3>Benefits</h3>
+				<p>
+					The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+				</p>
+				<ul>
+					<li>Schedule meetings for you</li>
+					<li>Excel automation to complete your work for you</li>
+					<li>Responds to emails on your behalf</li>
+					<li>Does all your work for you with our revolutionary Al technology</li>
+				</ul>
+			</div>
+			<div class="quote">
+				<div class="blockquote">
+					"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work."
+				</div>
+				<div class="quote-name">- Amanda, Intuit</div>
+			</div>
+			<div class="download-container">
+				<a class="button download-footer">Sign Up Now!</a>
+			</div>
+			<center>
```

Comment:
The ```<center>``` tag is deprecated (meaning it might not work in future versions) in favour of css centering